### ISSUE TITLE
Questions about your work

### ISSUE THREAD

**Issue by ERRIC729** [samhubs/Loco-for-Yolo#1]
Hi，Samhubs，First of all, thank you very much for your work, I am also using yolov8 for part of loco dataset, but I would like to ask about the original loco labeling is about 5593 images, but the training set you labeled is 5083, and the test set is 1436. How do you classify the number of images？

**Comment by samhubs** [comment]
Hey Sorry, this repo is obsoleted. I am closing the comment and the repo for now. My apologies for the inconveniece.
